packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -617,22 +618,80 @@ class _ConstraintsDescriptorState extends State<ConstraintsDescriptor>
 
 
 class StoryOfYourFlexWidget extends StatelessWidget {
-  const StoryOfYourFlexWidget(InspectorTreeNode node, {
+  StoryOfYourFlexWidget(this.node, {
     Key key,
   }) : super(key: key);
 
+  InspectorTreeNode node;
+
   @override
   Widget build(BuildContext context) {
+    final Map<String, Object> flexDetails = node.diagnostic?.json['flex'];
+    final int numOfChildren = node.diagnostic.childrenNow.length;
+    final List<Widget> children = node.diagnostic.childrenNow.map((RemoteDiagnosticsNode child) => Expanded(
+      flex: 1,
+      child: Container(
+        decoration: BoxDecoration(
+          color: Colors.green,
+          border: Border.all(
+            color: Colors.black,
+            width: 1.0,
+          ),
+          boxShadow: [
+            BoxShadow(
+              color: Colors.black,
+              offset: Offset.zero,
+              blurRadius: 10.0,
+            )
+          ],
+        ),
+        child: Center(
+          child: Text(child.description),
+        )
+      ),
+    )).toList();
+
+    Widget flexWidget = (flexDetails['direction'] as String).contains('horizontal') ?
+      Row(children: children) :
+      Column(children: children);
     return Dialog(
-      child: Column(
-        children: const [
-          Text('Story of Flex Layout page'),
-          Expanded(
-            child: Center(
-              child: Text('TODO'),
+      child: Container(
+        margin: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            Container(
+              margin: const EdgeInsets.only(bottom: 4.0),
+              child: const Text('Story of Your Flex Widget', style: TextStyle(
+                fontWeight: FontWeight.bold,
+                fontSize: 20.0,
+              )),
             ),
-          ),
-        ],
+            Center(
+              child: Text('${node.diagnostic?.json['flex']}'),
+            ),
+            Center(
+              child: Text('${node.diagnostic?.childrenNow}'),
+            ),
+            Expanded(
+              child: Container(
+                color: Colors.blue,
```

Comment:
try to use colors from the Theme whenever possible. That will make the ui work in dark and light theme.
See  calls to Theme.of in the code for examples.